src/config.zig

```diff
@@ -309,15 +312,26 @@ pub const configs = struct {
             @compileError("invalid release version");
         };
 
+        const release_client_min_raw = if (build_options.release_client_min) |release_client_min|
+            release_client_min
+        else
+            build_options.release;
+        const release_client_min_triple = vsr.ReleaseTriple.parse(release_client_min_raw) catch {
+            @compileError("invalid release_client_min version");
+        };
```

Comment:
Can you additionally assert:

```
assert((release_triple.value == vsr.ReleaseTriple.minimum.value) ==
    (release_client_min_triple.value == vsr.ReleaseTriple.minimum.value));
```

i.e.: If `release_triple` is _not_ the minimum (0.0.1) then `release_client_min_triple` is _also_ not the minimum.

Reply:
And imo we should pass 0.0.1 as the default in `build.zig`, rather than using `build_options.release` as the default. That way we can't accidentally publish 0.15.x with a minimum client version _equal to_ 0.15.x if we miss-set the parameter.

Reply:
Hmmm - I think the behavior we want here is for it to be an error to specify one of `release` / `release_client_min` without the other too?

Eg,
`-Dconfig-release=...` -> error
`-Dconfig-release=... -Dconfig-release-client-min=...` -> good
`-Dconfig-release-client-min=...` -> error
`` -> good (both are `0.0.1`)

Reply:
Yeah, that works too!